pom.xml

```diff
@@ -107,6 +107,11 @@
         <plugin>
           <groupId>org.apache.maven.plugins</groupId>
           <artifactId>maven-failsafe-plugin</artifactId>
+          <configuration>
+            <systemPropertyVariables>
+              <io.fluo.it.instance.name>it-instance-maven</io.fluo.it.instance.name>
```

Comment:
Was this added with the intention that eclipse would not set and would therefore end up using the default instance name?